src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1130,6 +1130,21 @@
      * The statement terminator ({@code ;}).  Depending on the
      * context, this make occur as a sibling, a child, or not at all.
      *
+     * <p>For example:</p>
+     * <pre>boolean var = true;</pre>
```

Comment:
please, make tags and content on separate lines, like in [here](https://github.com/checkstyle/checkstyle/pull/9378/files#diff-ba93830ba2cf23d542b30560fb43bdc8d8d1be05025e84ee6a96cca6ef8e3342R1430)

Reply:
@timurt done